### PR TITLE
Howtocon

### PR DESCRIPTION
**Pull request by MalloZup**
- improve some doc

### CODE REVIEW

File: doc/CONTRIBUTING.md [hunk 1]
```diff
@@ -3,9 +3,27 @@
...
+
+## Issues before big Prs aka features 
+
+If you want to contribute, just look at current issues and start to discuss/thinking how you will solve some of them.
+If you have in mind a new feature, just **open an issue first.**
+ 
+Instead of Proposing directly a Pull Request, openining an issue where you describe what you want to do, how, etc.,
+will give you directly a feedeback, and have a sense of knowing what you will do by describing, discussing it **before**, 
+will  increase the ease you will code ( discussing the design before, instead of changing it after).
+
```
Comment (juliogonzalez) [inline doc/CONTRIBUTING.md]: One empty line is enough to separate paragraphs :-)

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -3,9 +3,27 @@
...
+
+For small bugfixing/improvements of course you don't need an Issue referenced.
+
```
Comment (juliogonzalez) [inline doc/CONTRIBUTING.md]: And between sections (in the end GitHub will only consider one line when using markdown)

File: doc/CONTRIBUTING.md [hunk 3]
```diff
@@ -3,9 +3,27 @@
...
+
+
+## GIT Workflow
+
```
Comment (juliogonzalez) [inline doc/CONTRIBUTING.md]: And same here.

**Comment by MalloZup** [comment]
@juliogonzalez  have a look once you are free ! :cityscape:

**Comment by juliogonzalez** [comment]
LGTM and merging.

**Pull request merged**